solaris/utils/geo.py

```diff
@@ -729,6 +729,8 @@ def polygon_to_coco(polygon):
         coords = polygon.exterior.coords.xy
     elif isinstance(polygon, str):  # assume it's WKT
         coords = loads(polygon).exterior.coords.xy
+    elif isinstance(polygon, MultiPolygon):
```

Comment:
See comment on  `sol.data.coco` line 200: would be great to accommodate multipolygons, either in this function or in whatever calls it, by exploding them into separate geometries.

Reply:
Works for me!